src/gsy_e/models/strategy/energy_parameters/load.py

```diff
@@ -216,6 +222,14 @@ def update_energy_requirement(self, time_slot):
         self.state.set_desired_energy(load_energy_kwh * 1000, time_slot, overwrite=False)
         self.state.update_total_demanded_energy(time_slot)
 
+    def set_energy_measurement_kWh(self, time_slot: DateTime) -> None:
+        if gsy_e.constants.CONNECT_TO_PROFILES_DB:
+            measurement_from_profile_kWh = self.measurement_profile.profile.get(time_slot)
+            if measurement_from_profile_kWh is not None:
+                self.state.set_energy_measurement_kWh(measurement_from_profile_kWh, time_slot)
```

Comment:
I have concerns about this actually. I thought that the energy parameters would be the place to decide how to deal with missing datapoints, and I would prefer that the lack of measurements is set to a reasonable default. If we omit updating the state here, the simulation results service would have to deal with the gap/missing entry in the measurement profiles. 
Better to discuss this offline as well, but I see 2 options here:
- leave it as is, and have the simulation results service / library deal with the missing datapoints
- use 0 as a default, in order to not have gaps 
- a combination of 1 and 2, using None as a default value, and then have the simulation results service handle the None values by omitting them from profiles

Reply:
Thanks for this! I would vote for your third option, the combination of both.